packages/service/ni_measurement_plugin_sdk_service/measurement/info.py

```diff
@@ -5,7 +5,9 @@
 import enum
 from pathlib import Path
 from typing import NamedTuple
+from ni.measurementlink.discovery.v1.client import ServiceInfo
 
+__all__ = ["ServiceInfo"]
```

Comment:
This excludes MeasurementInfo, TypeSpecialization, and DataType, which hides them from some workflows/tooling.